apps/aecore/src/aec_db.erl

```diff
@@ -588,7 +575,11 @@ wait_for_tables(Tabs, Sofar, Period, Max) when Sofar < Max ->
             wait_for_tables(NotLoaded, Sofar + Period, Period, Max)
     end;
 wait_for_tables(Tabs, Sofar, _, _) ->
-    {timeout, Sofar, Tabs}.
+    lager:error("Tables not loaded after ~p minutes: ~p", [Sofar, Tabs]),
+    %% This is serious and user intervention needed. Stop the system instead 
+    %% of keeping retrying, but also raise an error for the crash log.
+    init:stop(),
+    error({tables_not_loaded, Tabs}).
```

Comment:
If `init:stop()` were synchronous, this would not be reached. But it is currently implemented as async, so this code is sometimes reached. And it is improvement from before. Ok.